follower/follower_builder.go

```diff
@@ -0,0 +1,716 @@
+package follower
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	upstream "github.com/onflow/flow-go/module/upstream"
```

Comment:
can you merge the imports into 3 groups: stdlib, 3rd party (stuff outside for onflow/flow-go), and `onflow/flow-go`

Reply:
Addressed: https://github.com/GetElastech/flow-go/commit/8ff48be52f60861ac00f32ec5f62092386e8edf3
I sorted the two groups and did the split making four.